### PR TITLE
RHDEVDOCS 6116 Release Notes for Pipelines 1.14.5

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

Please CP to pipelines-docs-1.14, pipelines-docs-1.15

Issue:

RHDEVDOCS 6116

Link to docs preview:

https://78937--ocpdocs-pr.netlify.app/openshift-pipelines/latest/about/op-release-notes.html#release-notes-1-14-5_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-6116

**Review by jkandasa — APPROVED** [review]
LGTM

**Review by ppitonak — APPROVED** [review]

**Review by vikram-raj — COMMENTED** [review]
LGTM

File: modules/op-release-notes-1-14.adoc [hunk 1]
```diff
@@ -350,3 +350,15 @@ With this update, {pipelines-title} General Availability (GA) 1.14.4 is availabl
...
+== Release notes for {pipelines-title} General Availability 1.14.5
+
+With this update, {pipelines-title} General Availability (GA) 1.14.5 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-5_{context}"]
+=== Fixed issues
+
+* Before this update, when you used the web console and clicked a pipeline in the overview page, the pipeline details page did not contain information about tasks in the pipeline. With this update, when you click a pipeline in the overview page, the pipeline details page contains the required information.
+
+* Before this update, when you configured Tekton Chains to disable storing OCI artifacts by setting an empty `artifacts.oci.storage` value in the `TektonConfig` CR, the configuration did not work and Tekton Chains attempted to store the artifacts and logged a failure in the `chains.tekton.dev/signed` annotation. With this update, when you set an empty `artifacts.oci.storage` value in the `TektonConfig` CR, Tekton Chains does not attempt to store OCI artifacts.
```
Comment (GroceryBoyJr) [inline modules/op-release-notes-1-14.adoc:364]:
```suggestion
* Before this update, when you configured {tekton-chains} to disable storing OCI artifacts by setting an empty `artifacts.oci.storage` value in the `TektonConfig` CR, the configuration did not work and {tekton-chains} attempted to store the artifacts and logged a failure in the `chains.tekton.dev/signed` annotation. With this update, when you set an empty `artifacts.oci.storage` value in the `TektonConfig` CR, {tekton-chains} does not attempt to store OCI artifacts.
```

Comment (GroceryBoyJr) [reply modules/op-release-notes-1-14.adoc:364]: @mramendi , Release Notes are not intended to be reusable content and reflect the current state of the item being discussed. However, using the AsciiDoc attribute MAY contribute to accuracy, so I suggest it. Totally your call, sir.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM except one nit.

**Pull request merged**